test/ConstructiveSolidGeometry/CSG_primitives.jl

```diff
@@ -278,4 +278,12 @@ no_translations = (rotation = one(SMatrix{3, 3, T, 9}), translation = zero(Carte
         CSG.ConeMantle()
         CSG.ConeMantle(φ=(1.,2.))    
     end
+    @testset "Point" begin
+        cart = @inferred CSG.CartesianPoint(x=2f0,z=1f0)
+        @inferred CSG.CartesianPoint{Float32}(x=2)
+        cyl = @inferred CSG.CylindricalPoint{Float32}(r=2.,z=1.)
+        cyl2 = @inferred CSG.CylindricalPoint(φ=3π)
+        @test CartesianPoint(cyl) == cart
+        @test cyl2.φ == T(π)
```

Comment:
Here, you could also check if the type conversion of `CartesianPoint` to `CylindricalPoint` works, something like
```julia
@test CartesianPoint(x = 1) = CartesianPoint(CylindricalPoint(r = 1))
```
(not sure if this works right of the box. though.